alacritty_terminal/tests/ref/grid_reset/config.json

```diff
@@ -1 +1 @@
-{"history_size":100}
+{"history_size":100}
```

Comment:
There's no change to this file, so we shouldn't update it?

Reply:
Well, alacritty produces a bit different format now, so it was just changed by `mv *.json alacritty_terminal/tests/ref/grid_reset/`. Maybe keeping new "format" make more sense. I can revert it if you insist though.

Reply:
I'd revert, just to keep the changes minimal. This change isn't necessary, so I'd rather not commit it.